packages/renderer/src/api/Component.ts

```diff
@@ -0,0 +1,5 @@
+export default interface Component {
```

Comment:
We have this type also in Workspace package - where do you think is the correct place to leave it?
(I think definitely not in both packages)

Reply:
right, it's just didn't merged yet, so I planning to replace it asap.